src/amuse/__init__.py

```diff
@@ -16,9 +16,9 @@
             a uniform interface to enable coupling
 
 Help is available for each of these packages, e.g.:
-> amuse.sh
->>> import amuse.ic
->>> help(amuse.ic) # doctest: +ELLIPSIS
+> ipython
+In [1]: import amuse.ic
+In [2]: help(amuse.ic) # doctest: +ELLIPSIS
```

Comment:
I don't think its necessary to use ipython for this...

Reply:
No, but it's just an example. Python works fine too. Just wanted to get rid of amuse.sh which we shouldn't be using anymore.

Reply:
fixed anyway.